ui/src/experiments/components/ExperimentItem.jsx

```diff
@@ -29,6 +34,19 @@ function ExperimentItem({ metadata }) {
           )}
         </>
       }
+      rightActions={
+        <ListItemAction>
+          <Link to={`${EXPERIMENTS}?q=${getPapersQueryString(recordId)}`}>
+            <IconText
```

Comment:
we could also extract this in a component which takes the literature query, we have it in multiple places already